README.md

```diff
@@ -81,6 +81,22 @@ Then, assuming a bucket named `wind-turbine` has been created through [MinIO's w
 modelardbd edge path_to_local_data_folder s3://wind-turbine
 ```
 
+`modelardbd` also supports using [Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs/) 
+for the remote object store. To use [Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs/), 
+set the following environment variables instead:
+
+```shell
+AZURE_STORAGE_ACCOUNT_NAME
+AZURE_STORAGE_ACCESS_KEY
+```
+
+`modelardbd` can then be run in edge mode with transfer of the ingested time series to the 
+[Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs/) container `wind-turbine`:
```

Comment:
Maybe explicitly if this assumes the container is created by the user like with MinIO or if it automatically created.

Reply:
Fixed.